CHANGELOG.md

```diff
@@ -26,6 +26,10 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/report/compare/release-3.11.5...master
 
+### Added
+- Add Verdict.Builder with all fields to make class usable in unit tests
```

Comment:
> all fields

It doesn't let you build all fields currently.

> to make class usable in unit tests

It's not usable now? A better description of the problem would be more useful for the consumers (either in changelog message or Jira ticket).